### ISSUE TITLE
An element could not be located on the page using the given search parameters.

### ISSUE THREAD

**Issue by ChengQiushuo** [microsoft/WinAppDriver#1532]
desired_caps = {}
         desired_caps["app"] = "Root"
         self.driver = webdriver.Remote(
              command_executor='http://127.0.0.1:4723',
              desired_capabilities=desired_caps
         )
Current my driver's handle is the desktop
And I can find other app's handle using:
appWindow = self.driver.find_element_by_name("appname")
appWindowHandle = appWindow.get_attribute("NativeWindowHandle")

except for the window 'NVIDIA uninstall'
And I can't find the button or label on this window directly.

In inspect.exe, 'NVIDIA uninstall' is the children of the desktop just like other apps, but the exception is thrown when I run my code.

**Comment by anunay1** [comment]
are the buttons visible in inspect.

**Comment by ChengQiushuo** [comment]
> are the buttons visible in inspect.

thanks for your quick response 

yes it is visible in inspect
I can see the name, class name, automation Id and everything

**Comment by anunay1** [comment]
Can you share code?

**Comment by ChengQiushuo** [comment]
> Can you share code?

Yes of course, but some of the names of element are shown in Chinese. I will add some translation in the annotation  to make it easier for you to understand.

class Tests(unittest.TestCase):
    @classmethod
    def setUpClass(self):
         # set up appium
         desired_caps = {}
         desired_caps["app"] = "Root"
         self.driver = webdriver.Remote(
              command_executor='http://127.0.0.1:4723',
              desired_capabilities=desired_caps
         )
#open the control.exe because if I use desired_caps["app"] = "the executable path for control.exe", the driver cannot find the application 
         os.popen('control.exe')

    @classmethod
    def tearDownClass(self):
       self.driver.quit()

    def test_uninstall_cuda_documentation(self):
#uninstall application interface where all install application are shown in a lisst in control.exe
        self.driver.find_element_by_name("卸载程序").click() 
#double click for uninstall to start
        uninstall_element = self.driver.find_element_by_name("NVIDIA CUDA Documentation 10.0")
        uninstallAction = selenium.webdriver.common.action_chains.ActionChains(self.driver)
        uninstallAction.move_to_element(uninstall_element)
        uninstallAction.double_click(uninstall_element)
        uninstallAction.perform()
#find the uninstall button, "卸载" means "uninstall"
        while len(self.driver.find_elements_by_name("卸载（C)")) == 0:
            time.sleep(1)
        self.driver.find_element_by_name("卸载（C)").click()

Then my program is stuck in the while loop.
These code works fine when uninstalling python 3.6. But when I try to uninstall CUDA, the elemnet could not be found.

**Comment by ChengQiushuo** [comment]
@anunay1  When the uninstall .exe running, a splash windows appears firsts, I wonder if that's the reason why I can't seems to find any element or the window itself.

**Comment by anunay1** [comment]
Yes that's the reason you are not able to find any controls.

**Comment by ChengQiushuo** [comment]
@anunay1 But I tried creating a new session after the splash window disappears, desired_caps["app"] = "Root" , and I still can find every other window expect for the NIVIDA uninstall window.  Any pointers?

**Comment by ChengQiushuo** [comment]
I found this in some similar issue, but I don't know how to code it in Python.

https://github.com/microsoft/WinAppDriver/blob/5398a93938747305c60596a55502bdad0319a7cd/Tests/WebDriverAPI/Window.cs#L98
